src/LibraryManager.Vsix/UI/Models/InstallDialogViewModel.cs

```diff
@@ -338,8 +340,56 @@ private bool CanInstallPackage()
                 return false;
             }
 
+            return true;
+        }
+
+        public string ErrorMessage
+        {
+            get { return _errorMessage; }
+            set { Set(ref _errorMessage, value); }
+        }
+
+        public bool IsLibraryInstallationStateValid()
```

Comment:
>public [](start = 8, length = 6)

Is this used outside this class?

Reply:
Yes this is Used in InstallDialog to determine if we want to display message box or close the dialog when Install button is clicked.

---
In reply to: [201159105](https://github.com/aspnet/LibraryManager/pull/293#discussion_r201159105) [](ancestors = 201159105)